helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -352,27 +353,28 @@ public void enableInstance(final String clusterName, final String instanceName,
         clusterName);
     BaseDataAccessor<ZNRecord> baseAccessor = new ZkBaseDataAccessor<>(_zkClient);
     enableSingleInstance(clusterName, instanceName, enabled, baseAccessor, disabledType, reason);
```

Comment:
Shall we add some comments here saying "right now is transition period so both single and batch are enabled, should be removed once migration is completed balabala"?

Reply:
TFTR. Updated